test/system/roles/public_health/dashboard/export_verifier.rb

```diff
@@ -30,9 +30,9 @@ def verify_sara_alert_format(user_label, workflow)
 
   def verify_excel_purge_eligible_monitorees(user_label)
     current_user = @@system_test_utils.get_user(user_label)
-    download_comprehensive_export_files(current_user, 'full_history_purgeable')
+    download_full_history_export_files(current_user, 'full_history_purgeable')
     xlsx_monitorees = get_xlsx('Sara-Alert-Purge-Eligible-Export-Monitorees-????-??-??T??_??_?????_??-?.xlsx')
-    xlsx_assessments = get_xlsx('Sara-Alert-Purge-Eligible-Export-Assessments-????-??-??T??_??_?????_??-?.xlsx')
+    xlsx_assessments = get_xlsx('Sara-Alert-Purge-Eligible-Export-Reports-????-??-??T??_??_?????_??-?.xlsx')
```

Comment:
I wonder if some sort of export filename helper might be useful. Would also be an opportunity to ensure all filenames are sanitized in a central location.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/562/commits/18f2cd1c1295b23fca3eb469bf542092217c6acb#diff-05cdaff5cea543df19a21cfeba003c0707f912431419e655445b2d3a9d47667eR19